x-pack/plugin/blob-cache/src/test/java/org/elasticsearch/blobcache/shared/SharedBlobCacheServiceTests.java

```diff
@@ -1010,13 +1044,16 @@ public void execute(Runnable command) {
                     cacheKey,
                     randomIntBetween(0, 10),
                     randomLongBetween(1L, regionSize),
-                    (channel, channelPos, streamFactory, relativePos, length, progressUpdater) -> {
-                        throw new AssertionError("should not be executed");
-                    },
+                    (channel, channelPos, streamFactory, relativePos, length, progressUpdater, completionListener) -> completeWith(
+                        completionListener,
+                        () -> {
+                            throw new AssertionError("should not be executed");
+                        }
+                    ),
                     bulkExecutor,
                     future
                 );
-                assertThat("Listener is immediately completed", future.isDone(), is(true));
+                assertThat("Listener is immediately completionListener", future.isDone(), is(true));
```

Comment:
nit String edit is not that readable. Maybe original string is good enough? (similar comment for the other cases where the same string is edited)

Reply:
Fixed, thank you! That was an unintended change.